<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="headerPanel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
    <widget class="QLabel" name="headerLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>TRANSFER OWNERSHIP</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="footPanel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>600</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QWidget" name="footPanel1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>70</height>
      </rect>
     </property>
     <widget class="QPushButton" name="transferOwnershipBut">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>198</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>TRANSFER OWNERSHIP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="outgoingTransfersBut">
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>2</y>
        <width>198</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>OUTGOING TRANSFERS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="incomingTransfersBut">
      <property name="geometry">
       <rect>
        <x>401</x>
        <y>2</y>
        <width>198</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>INCOMING TRANSFERS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="footPanel2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>600</width>
       <height>70</height>
      </rect>
     </property>
     <widget class="QPushButton" name="finalSaleBut">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>132</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>FINAL SALE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnedItemsBut">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>2</y>
        <width>132</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>RETURNED ITEMS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="forensicsBut">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>2</y>
        <width>132</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>FORENSICS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="activateItemsBut">
      <property name="geometry">
       <rect>
        <x>401</x>
        <y>2</y>
        <width>131</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>ACTIVATE ITEMS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adminBut">
      <property name="geometry">
       <rect>
        <x>533</x>
        <y>2</y>
        <width>66</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
selection-background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>ADMIN</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="POSTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>40</y>
      <width>596</width>
      <height>371</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QComboBox" name="recipientCombo">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>490</y>
      <width>596</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Arial&quot;;</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::NoInsert</enum>
    </property>
    <item>
     <property name="text">
      <string>Select/Enter Recipients</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="transferOwnershipActBut">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>570</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
border-left-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TRANFER OWNERSHIP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelOwnershipTransferActBut">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>570</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelOutgoingActBut">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>490</y>
      <width>596</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL TRANSFER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelIncomingActBut">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>490</y>
      <width>596</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL TRANSFER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerSaleActBut">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
border-left-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>REGISTER SALE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finalSaleCancelActBut">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnsActBut">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
border-left-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>REGISTER SALE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnsCancelActBut">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportCounterfeitActBut">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
border-left-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>REPORT COUNTERFEIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forensicsDoneActBut">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="activateItemsActBut">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
border-left-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ACTIVATE ITEMS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelActivateActBut">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>490</y>
      <width>295</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QFrame" name="returnedItemsFrame">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>40</y>
      <width>471</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="returnedItemsLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Receipt Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="returnedItemsLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>130</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Pin:</string>
     </property>
    </widget>
    <widget class="QLabel" name="returnedItemsReceiptLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>FSUFU58576HUJNEK</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pinLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>270</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="returnedItemsOKBut">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>115</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);
border-left-color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QFrame" name="forensicsFrame1">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>40</y>
      <width>596</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="forensicsLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Receipt Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="forensicsReceiptLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>FSUFU58576HUJNEK</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="forensicsFrame2">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>90</y>
      <width>596</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="forensicsLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
    <widget class="QLabel" name="forensicsLabel3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Current Owner:</string>
     </property>
    </widget>
    <widget class="QLabel" name="forensicsLabel4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Final Seller:</string>
     </property>
    </widget>
    <widget class="QLabel" name="forensicsDescriptionLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>PEANUT BUTTER</string>
     </property>
    </widget>
    <widget class="QLabel" name="forensicsCurrentOwnerLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>COMPANY X ENTERPRISES</string>
     </property>
    </widget>
    <widget class="QLabel" name="forensicsFinalSellerLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cancelActivateActBut_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelActivateActBut_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 170);
border-left-color: rgb(225, 225, 225);
border-top-color: rgb(225, 225, 225);
border-right-color: rgb(90, 90, 90);
border-bottom-color:rgb(90, 90, 90);
font: 75 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>POSTableView</class>
   <extends>QTableView</extends>
   <header>postableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
